modules/uninstall-cluster-logging-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-cluster-logging-operator_{context}"]
+= Uninstalling the {logging}
+
+You can stop log aggregation by deleting the {clo} and the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the *Administrator* perspective of the {product-title} web console.
+
+.Procedure
+
+. Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *ClusterLogging*.
+
+. On the *Custom Resource Definition Details* page, click *Instances*.
+
+. Click the options menu {kebab} next to the instance, then click *Delete ClusterLogging*.
+
+. Navigate to the *Administration* -> *Custom Resource Definitions* page.
+
+. Click the options menu {kebab} next to *ClusterLogging* and select *Delete Custom Resource Definition*.
++
+[NOTE]
+====
+Deleting the `ClusterLogging` CR does not remove the persistent volume claims (PVCs). To delete the remaining PVCs, persistent volumes (PVs), and associated data, you must take further action. Releasing or deleting PVCs can delete PVs and cause data loss.
+====
+
+. If you have created a `ClusterLogForwarder` CR, click the options menu {kebab} next to *ClusterLogForwarder*, then click *Delete Custom Resource Definition*.
```

Comment:
This part isn't consistent with the rest of the document in terms of deleting the instance first, then the CRD (like it is mentioned for `LokiStack` `ClusterLogging` and `Elasticsearch` instances).
It is technically correct though, deleting the CRD will delete its instances. I am only mentioning it for the sake of consistency.